README.md

```diff
@@ -39,6 +39,16 @@ Instructions for using Mythril Classic are found on the [Wiki](https://github.co
 
 For support or general discussions please join the Mythril community on [Discord](https://discord.gg/E3YrVtG).
 
+## Bulding the Documentation
+Mythril Classic's documentation is contained in the `docs` folder. It is based on Sphinx and can be built using the Makefile contained in the subdirectory:
```

Comment:
It might be good to mention that this process requires sphinx. what do you think?